slides/02_Setup.html

```diff
@@ -247,7 +247,16 @@ <h2>NPM scripts</h2>
             <p><span class="code">npm run dev</span></p>
             <p>
               Starts the app in dev mode with a hot reload server. The build
-              will fail if you have TypeScript errors.
+              will <strong>not</strong> run typescript checks.
+            </p>
+            <p>
+              Integrate typescript checking in the dev mode with the
```

Comment:
I suggest formulating this a bit softer – Vite explains here why type checking is not performed in dev mode: https://vitejs.dev/guide/features.html#transpile-only – I think it's good to show how it can be done. But people might not need it, as the IDE shows errors typically and if something is missed there is still type checking on the CI.